src/components/Tag/Tabs/VulnerabilitiesDetails.jsx

```diff
@@ -172,6 +243,10 @@ function VulnerabilitiesDetails(props) {
     };
   }, []);
 
+  useEffect(() => {
```

Comment:
do we want the call to getAllCves to be in a naked useEffect?  I'm thinking it would maybe be better to do it after the user clicks on export, to avoid the performance hit.